command/host_create.go

```diff
@@ -23,6 +25,10 @@ func (c *createHost) Name() string { return "host.create" }
 
 func (c *createHost) ParseParams(params map[string]interface{}) error {
 	c.CreateHost = &apimodels.CreateHost{}
+
+	if params["background"] == nil {
```

Comment:
you probably want to check the second value (e.g. _, ok := params["background"]; ok) rather than the value.